css-values-4/Overview.bs

```diff
@@ -2928,6 +2928,57 @@ Functional Notations</h2>
 	Other [=functional notations=] are defined in their own modules;
 	for example the <<absolute-color-function>>s are defined in [[CSS-COLOR-5]].
 
+<h3 id="fuctional-serialization">
+Serialization of Functional Notations</h3>
+
+	<div algorithm="serialize a functional notation">
+		Unless otherwise specified,
+		to <dfn export>serialize a functional notation</dfn>:
+
+		1. Let |s| be a string containing
+			the function's name in lowercase,
+			followed by “(” (U+0028 LEFT PARENTHESIS).
+
+		2. Append to |s|
+			the serializations of the arguments of the [=functional notation=]
+			per their individual grammars,
+			in the order the grammars are written in,
+			joining space-separated tokens with a single space,
+			and following each serialized comma with a single space (U+0020 SPACE).
```

Comment:
Is it possible to describe here that other tokens like `;`, `/` and so on are also followed by a single space? Or is there no consistency in what we have now?